rotkehlchen/api/v1/schemas.py

```diff
@@ -616,6 +616,10 @@ class HistoryEventSchema(DBPaginationSchema, DBOrderBySchema):
         SerializableEnumField(enum_class=HistoryBaseEntryType),
         load_default=None,
     )
+    excluded_entry_types = DelimitedOrNormalList(
```

Comment:
Yes indeed, I like your suggestion. Do you know if we already have a similar behaviour (include or exclude for a field) in other api endpoints, so that we can follow one common approach in such cases? If there isn't one already, then perhaps we can use what you describe here.

Reply:
I don't think we do any such thing. It does not sound bad. Good idea probably. But we would need to add new code for this as I don't think we have it anywhere else.

Could eventually abstract it into a field type, if we add it to more places.